translations/strings.yaml

```diff
@@ -48,3 +48,7 @@
 # Remember! When you change this file, you must run the
 # `./scripts/generate_strings.py` script to regenerate the string files.
 #
+
+serversView:
```

Comment:
You probably want a review from @bakulf here, as I don't know what kind of naming structure he wanted to use, and if this matches his expectations.

Reply:
thank you @flodolo , just tagged @bakulf for review as I'm not sure about this either.